posts/start-web-performance-first-steps.md

```diff
@@ -0,0 +1,120 @@
+
+> **Better audits and CI are our greatest tools for automatic knowledge sharing**
+
+We're still dillengently working in our audits and CI pipeline and far from perfect, but our performance tests currently consists of:
+
+1. [Lighthouse CI Performance Budgets](https://web.dev/use-lighthouse-for-performance-budgets/): We use these to prevent increases in Javascript size, Image Size, Requests Counts and Total Size of each of our main entry points -- Our [Homepage](http://quintoandar.com.br/), our [Search Page](http://quintoandar.com.br/alugar/imovel/sao-paulo-sp-brasil), and our [Listing Page](https://www.quintoandar.com.br/imovel/892879684). This is our most important audit and the one that gives us more safety, but the others allows us to better understand the issues;
+2. [Danger](https://github.com/danger/danger) script for file sizes: While the most important think is measuring the impact in each route separately, distinct file sizes also impact our users in different ways, so we track the sizes on our must important files, either because they're very common on different pages or why they're already big, and try not to increase them too often;
+
+We're currently working on adding more audits and experimenting new ones. Since we're directly impacting how code is shipped in the company, before actually being used widely we must first guarantee that each audit:
+
+1. **Does not** give false negatives;
+2. **Minimizes** false positives: This is the goal, but the former is a restriction. If an audit gives too many false negatives, people will lose faith in its usefullness. This is worse than not having the audit since it's wasting developer time and patience.
+
+## Step Two: Set up a Group
+
+## Step Three: Make performance a part of the Company's Culture
+
+---
+
+## In Summary
+
+- The sooner you start working on your product's performance, the better;
+- Knowledge and engagement are the most important challenges to solve so focus on that;
+- Performance constraints will raise the bar for your engineers and provide great value for your users, make sure everyone understand that and the cycle may be self-reinforcing;
+- Measure and Lock: Use real user metrics and lab metrics to prevent degradation, to understand how performance affects your business and to pass knowledge forward;
+- Set up a group: Gather individuals that are interested in the subject and give them support and incentive to keep the discussion alive, evolve and maintain the tools;
+- Make performance a part of the Company's Culture: Build tools to lower the barriers and incetivize the Engineering team, share learnings and results to engage the product, design and business areas of the company
+
```

Comment:
How about a small paragraph about the section article

"Next article I'll deep dive in tooling, how we tested and choose the right tools for measuring, locking and debugging performance" (for example)